website/docs/cloud-docs/integrations/kubernetes/setup.mdx

```diff
@@ -58,20 +60,46 @@ The HCP Terraform Operator for Kubernetes supports the following versions:
    kubectl -n $NAMESPACE create secret generic workspacesecrets --from-literal=secret_key=abc123
    ```
 
-1. Install the [HCP Terraform Operator for Kubernetes with Helm](https://github.com/hashicorp/terraform-helm).
+1.  Add the HashiCorp Helm repository.
 
-   ```
-   helm repo add hashicorp https://helm.releases.hashicorp.com
+    ```
```

Comment:
I'm so sorry to be annoying, and I know this wasn't in the original PR, but would you mind adding a language to all the codeblocks in the file? I believe it'd be `shell-session` for all of them, and it'll really help the syntax of the TFE commands pop.